pkg/backend/diy/state.go

```diff
@@ -337,6 +337,10 @@ func (b *diyBackend) removeStack(ctx context.Context, ref *diyBackendReference)
 // backupTarget makes a backup of an existing file, in preparation for writing a new one.
 func backupTarget(ctx context.Context, bucket Bucket, file string, keepOriginal bool) string {
 	contract.Requiref(file != "", "file", "must not be empty")
+	if ok, err := bucket.Exists(ctx, file); !ok && err == nil {
```

Comment:
Nit but I feel like it's "OK" for the file not to exist, so maybe another name?
```suggestion
	if exists, err := bucket.Exists(ctx, file); !exists && err == nil {
```

Reply:
Yeah, I think that's nicer, thanks!